BUILD.bazel

```diff
@@ -68,3 +69,15 @@ filegroup(
     ],
     visibility = ["//visibility:public"],
 )
+
+bzl_library(
+    name = "bzl",
```

Comment:
Is there a naming convention for these rules? This doesn't seem descriptive, but I guess if it's just a set of all .bzl files in each package, then it's good to use the same name. Just want to make sure that's what other rulesets are doing.

Reply:
Let's update this to match the convention from `@bazel_skylib//gazelle/bzl`.